velox/functions/sparksql/Register.cpp

```diff
@@ -402,6 +402,8 @@ void registerFunctions(const std::string& prefix) {
       {prefix + "unix_millis"});
   registerUnaryIntegralWithTReturn<MillisToTimestampFunction, Timestamp>(
       {prefix + "timestamp_millis"});
+  registerUnaryFloatingWithTReturn<SecondsToTimestampFunction, Timestamp>(
+        {prefix + "timestamp_seconds"});
```

Comment:
We also need to document this function in https://github.com/facebookincubator/velox/tree/main/velox/docs/functions/spark, thanks.